tests/table/test_metaclass.py

```diff
@@ -109,6 +109,19 @@ class Classified(Table):
             Classified._meta.secret_columns, [Classified.top_secret]
         )
 
+    def test_secret_columns_with_argument(self):
+        """
+        Make sure TableMeta.secret_columns are setup correctly
+        with ``secret=True`` argument.
+        """
+
+        class Classified(Table):
+            top_secret = Varchar(secret=True)
```

Comment:
As a sanity check, add a non-secret column here too, to make sure only secret columns are added to `_meta.secret_columns`. For example:

```python
class Classified(Table):
    top_secret = Varchar(secret=True)
    public = Varchar()
```

You might also consider merging this test with the above test, so we have this:

```python
def test_secret_columns(self):
    """
    Make sure ``TableMeta.secret_columns`` are setup correctly.
    """

    class Classified(Table):
        top_secret = Secret()
        confidential = Varchar(secret=True)
        public = Varchar()

    self.assertEqual(
        Classified._meta.secret_columns,
        [Classified.top_secret, Classified.confidential]
    )